scripts/coverage_report_jenkins.py

```diff
@@ -473,8 +475,8 @@ def get_eligible_builds(jenkins_data, jenkins_job, cfme_version):
         cfme_version:  Versuion CFME sources this coverage is against.
```

Comment:
Looks good except this typo here in 'version'.
cfme_version:  Version CFME sources this coverage is against.